src/templates/layouts/staffProfile/dataService.tsx

```diff
@@ -5,7 +5,7 @@ import {
 } from '@/types/dataTypes/drupal/paragraph'
 import { StaffProfile, StaffProfileProps } from './index'
 import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
-import { mediaImageDataService } from '@/templates/common/media/dataService'
+import { generalEntityDataService } from '@/data/delegators/generalEntityDataService'
```

Comment:
can this be removed entirely? doesn't look like it's used in the file from what i can see